ods_ci/tests/Resources/Files/pipeline-samples/v2/dspa/dspa-template.yaml

```diff
@@ -29,7 +29,7 @@ parameters:
   value: "v2"
   name: DSP_VERSION
 - description: Object Storage Bucket Name
-  value: "ods-ci-ds-pipelines"
+  value: "test-bucket"
```

Comment:
Hi @jgarciao why did you change this bucket name?

Reply:
The real value is set in keyword `CreatePipelineServer`, this is just an example